src/main/java/com/example/usermanagementsystemAPIgateway/controller/APIController.java

```diff
@@ -18,14 +18,13 @@
 
     HttpHeaders headers = new HttpHeaders();
 
-    private String url = "http://localhost:8081/users/";
+    @Value("${user-management-system.url}")
+    private String url;
 
     @CrossOrigin
     @PostMapping(value = "/api/users")
```

Comment:
why do we need `api` in `/api/users/`. why is it not just `/users/`?

Reply:
Yes we can.
The same links are used in user-management-system . And I just wanted to highlight the difference between user-management-system and user-management-api-gateway as a API gateway.

Reply:
The same links are used in user-management-system . And I just wanted to highlight the difference between user-management-system and user-management-api-gateway as a API gateway.
In fact, if it has a specific cause
I do not know.